src/components/CurrentTours/CurrentTourCard.tsx

```diff
@@ -1,9 +1,21 @@
 import React from 'react'
+import './CurrentTours.css'
 
-export default function CurrentTourCard() {
+interface Props {
+  date: string;
+  location: string;
+}
```

Comment:
We should agree on a style and stick to it, but personally I think we should preface the `Prop` interfaces with the name of the component they're for i.e. `CurrentTourProps`. I think it reads a little better, even though it should be obvious based on what file your in, but most tutorials do just use `Props`. I might just be bitter about how everything had the same name in Crate.

Reply:
I agree with this Boone, that makes things more readable and good extra visual of what is what. Let's definitely plan on making that a norm.